bin/cmd.js

```diff
@@ -75,9 +77,12 @@ argv.http = allFalsy || argv.http
 argv.udp = allFalsy || argv.udp
 argv.ws = allFalsy || argv.ws
 
+argv['no-stats'] = !!argv['no-stats']
```

Comment:
This shouldn't be necessary. We can solve this with the default option to `minimist`. I'll fix it in a follow up commit, and you can see how to use it.